src/tests/unit/tests/DetailsView/components/narrow-mode-detector.test.tsx

```diff
@@ -6,19 +6,29 @@ import {
     NarrowModeDetector,
     NarrowModeDetectorProps,
     NarrowModeStatus,
-    narrowModeThresholds,
 } from 'DetailsView/components/narrow-mode-detector';
+import { NarrowModeThresholds } from 'DetailsView/components/narrow-mode-thresholds';
 import { shallow } from 'enzyme';
 import * as React from 'react';
 
 const TestComponent = NamedFC<{ narrowModeStatus: NarrowModeStatus }>('TestComponent', props => {
     return <h1>Test component</h1>;
 });
 
+const narrowModeThresholds = {
+    collapseHeaderAndNavThreshold: 600,
+    collapseCommandBarThreshold: 960,
+};
+
+const getNarrowModeThresholds = (): NarrowModeThresholds => {
+    return narrowModeThresholds;
```

Comment:
While this works out, we usually do use Mocks here. If we want to keep it this way, I would suggest naming them as mocks appropriately.